sw/device/lib/testing/entropy_testutils.c

```diff
@@ -57,3 +57,15 @@ void entropy_testutils_boot_mode_init(void) {
   setup_csrng();
   setup_edn();
 }
+
+void wait_entropy_src_state(dif_entropy_main_fsm_t state) {
+  dif_entropy_main_fsm_t cur_state;
+
+  dif_entropy_src_t entropy_src;
+  CHECK_DIF_OK(dif_entropy_src_init(
+      mmio_region_from_addr(TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR), &entropy_src));
```

Comment:
Do you want to call this here? since this is already called in `setup_entropy_src()` above? Maybe this should just take the handle as a parameter and this testutil could be used in conjunction with the `setup_entropy_src()` testutil or by calling the init DIF in the test itself.

Reply:
yeah you are right, admittedly i just got lazy :)
I should pass the handle in.

Reply:
fixed.